### ISSUE TITLE
Increasing Web-Editor Keyboard Accessibility 

### ISSUE THREAD

**Issue by kaiserarg** [processing/p5.js-web-editor#3077]
### Increasing Access

1. Adding more keyboard accessibility to p5.js’ web editor will increase access by allowing users who cannot use a mouse and/or a trackpad to navigate and interact with p5.js’ editor.
2. Increases user’s comfort when using the web editor by allowing them to customize their experience through shortcuts
3. Improve website usability when only using the keyboard

### Feature enhancement details

1. Problem

Lack of options to increase ease of access for individualized experience on p5.js’ web editor. For example, navigation keys are pre-binded and certain navigational features are only limited to mouse/trackpad.

2. Possible Solution:

We could provide default keybinds for all actions that can be undertaken on the website, and add customizable keyboard shortcuts for users to better personalize their experience with the web editor.

Let me know what you guys think and if this is a good QOL feature enhancement. I would love to work on this!
